libs/ui/src/admin/ArtApprovalModel/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,243 @@
+        // size={['xs', 'sm', 'md', 'lg', 'xl', 'full']}
+        size={['1088px']}
+        isOpen={isOpen}
+      >
+        <ModalOverlay />
+        <ModalContent>
+          <ModalCloseButton />
+          <ModalBody>
+            <Stack direction={['column', 'row']} spacing={4}>
+              <Stack w={'full'}>
+                {art.images && art.images.length > 1 ? (
+                  <Splide>
+                    {art?.images.map(image => (
+                      <Center as={SplideSlide} key={image.id}>
+                        <ArtImage image={image} alt={art.title} />
+                      </Center>
+                    ))}
+                  </Splide>
+                ) : (
+                  <ArtImage image={art.images?.[0]} alt={art.title} />
+                )}
+              </Stack>
+              <VStack spacing={4} align={'start'} justify="space-between">
+                <VStack align={'start'} justify="space-between">
+                  <Flex align="start" justify="start" w="full" h={8}>
+                    <Text color={'blue.400'} fontWeight={'bold'}>
+                      {art?.title}
+                    </Text>
+                  </Flex>
+                  <HStack spacing={3} w={'full'}>
+                    {/* TODO art owner avatar should be here*/}
+                    <Avatar
+                      w={8}
+                      h={8}
+                      src="https://bit.ly/dan-abramov"
+                    ></Avatar>
+                    <Text fontWeight={'bold'}>{art?.artist?.name}</Text>
+                  </HStack>
+                  <Flex align="start" justify={'start'} w="full" h={'72px'}>
+                    {/* TODO scroll should be here when art has long text*/}
+                    <Text>{art?.content}</Text>
+                  </Flex>
+                </VStack>
+
+                <VStack align={'start'} w={'full'}>
+                  <Flex align="start" justify={'start'} w="full">
+                    <Text color={'black'} fontWeight={'bold'}>
+                      Give Feedback
+                    </Text>
+                  </Flex>
+
+                  <HStack w="full">
+                    {/* avatar*/}
+                    <Stack
+                      align={'start'}
+                      h={'136px'}
+                      flex={1}
+                      textAlign="center"
+                    >
+                      <Avatar
+                        w={8}
+                        h={8}
+                        src="https://bit.ly/dan-abramov"
+                        // src={user?.avatar}
+                      ></Avatar>
+                    </Stack>
+                    <VStack w="full">
+                      {/* text area, button group*/}
+                      <Stack w={'full'}>
+                        {/* text area*/}
+                        <Textarea
+                          size={['sm', 'md', 'lg', 'xl']}
+                          onChange={handleInputChange}

```

Comment:
You should store `feedback` note in state to be able to send it through `onApprove` or `onReject`

```suggestion
                          onChange={e => setFeedback(e.target.value)}

```